alvr/sockets/src/packets.rs

```diff
@@ -97,13 +97,7 @@ pub enum ClientControlPacket {
 // legacy video packet
 #[derive(Serialize, Deserialize, Clone)]
 pub struct VideoFrameHeaderPacket {
-    pub packet_counter: u32,
     pub tracking_frame_index: u64,
-    pub video_frame_index: u64,
-    pub sent_time: u64,
-    pub frame_byte_size: u32,
-    pub fec_index: u32,
-    pub fec_percentage: u16,
 }
```

Comment:
This can be simplified with a single `Duration` (also the term "index" is leftover from legacy code, that is in fact a timestamp in nanoseconds).

Reply:
What do you mean exactly by simplifying? SocketSender requires data packet headers, we cannot just send `Duration`, and sending non-primitive type doesn't sound like a good idea.

Reply:
Here i meant that the heaser IS just a `Duration`. You can remove the wrapping struct.

Reply:
pretty sure this is solved but I'll let you resolve this one!